File: HA4_1.java
//Write a program which accept name from user and print that name.
//Input : Maitreya Gangurde  Output : Maitreya Gangurde

import java.util.*;

public class HA4_1 
{
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter Your Name");
        String iValue = sobj.nextLine();

        System.out.println("Your Name is " +iValue);
    
        sobj.close();
    }    
}

File: HA4_1.c
//Write a program which accept name from user and print that name.
//Input : Maitreya Gangurde  Output : Maitreya Gangurde
#include <stdio.h>

int main()
{
    char Name[20];

    printf("Please enter full name \n");
    scanf("%[^\n]", Name);

    printf("Your name is : %s\n", Name);

    return 0;
}